test/integration/deep/norelay/testdata/client.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: client
+spec:
+  selector:
+    matchLabels:
+      app: client
+  template:
+    metadata:
+      labels:
+        app: client
+    spec:
+      containers:
+      - name: curl
+        image: curlimages/curl
+        command: [ "sh", "-c", "--" ]
+        args: [ "while true; do sleep 10; done;" ]
```

Comment:
tioli

```suggestion
        args: [ "sleep", "infinity" ]
```

Reply:
Oh that's deep...

Reply:
... but it ended up not being supported in the cURL image :man_shrugging: